content/en/docs/instrumentation/cpp/manual.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Manual Instrumentation
+linkTitle: Manual
+weight: 3
+---
+
+Manual instrumentation is the process of adding observability code to your
+application.
+
+## Initializing tracing
+
+```cpp
+auto provider = opentelemetry::trace::Provider::GetTracerProvider();
+auto tracer = provider->GetTracer("foo_library", "1.0.0");
+```
+
+The `TracerProvider` acquired in the first step is a singleton object that is
+usually provided by the OpenTelemetry C++ SDK. It is used to provide specific
+implementations for API interfaces. In case no SDK is used, the API provides a
+default no-op implementation of a `TracerProvider`.
+
+The `Tracer` acquired in the second step is needed to create and start Spans.
+
+## Start a span
+
+```cpp
+auto span = tracer->StartSpan("HandleRequest");
+```
+
+This creates a span, sets its name to `"HandleRequest"`, and sets its start time
+to the current time. Refer to the API documentation for other operations that
+are available to enrich spans with additional data.
+
+## Mark a span as active
+
+```cpp
+auto scope = tracer->WithActiveSpan(span);
+```
+
+This marks a span as active and returns a `Scope` object. The scope object
+controls how long a span is active. The span remains active for the lifetime of
+the scope object.
+
+The concept of an active span is important, as any span that is created without
+explicitly specifying a parent is parented to the currently active span. A span
+without a parent is called root span.
+
+## Create nested Spans
+
+```cpp
+auto outer_span = tracer->StartSpan("Outer operation");
+auto outer_scope = tracer->WithActiveSpan(outer_span);
+{
+    auto inner_span = tracer->StartSpan("Inner operation");
+    auto inner_scope = tracer->WithActiveSpan(inner_span);
+    // ... perform inner operation
+    inner_span->End();
+}
```

Comment:
Is the brace scoping necessary here, or is it convention in C++, or is it just illustrative?

Reply:
The brace scoping is required to manage the lifetime of `inner_scope` object.